Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu
 ENV NODE_ENV production
 
 RUN apt-get -y update && apt-get -y install \
```

Comment:
please change `FROM` to `FROM node:5.10.0` otherwise this will break. also, please pull to one line.